usr/share/rear/prep/COMMVAULT/default/400_prep_galaxy.sh

```diff
@@ -0,0 +1,13 @@
+#
+# prepare stuff for Commvault
+#
+
+COPY_AS_IS+=( "${COPY_AS_IS_COMMVAULT[@]}" )
+COPY_AS_IS_EXCLUDE+=( "${COPY_AS_IS_EXCLUDE_COMMVAULT[@]}" )
+
+PROGS+=( chgrp touch )
+
```

Comment:
Are 'chgrp' and 'touch' optional?
If not better use REQUIRED_PROGS.

Reply:
yes